source/open-source/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Open Source
+date: 2025-03-18
+layout: open-source
+
+# Sections schema
+# - title: "Section title"
+#   description: "Section description"
+#   items:
+#     - name: "Project title"
+#       description: "Project description to put in the body of the card"
+#       github_url: "GitHub repository link"
+#       url: "Optional: Link to open when the project card is clicked; falls back to `github_url` if omitted"
+#       logo: "Optional: Link to logo file"
+#       logo_scale: "Optional: will be added as CSS height: `logo_scale`"
+
+sections:
+  - title: "Energy-Efficient Systems"
+    description: "Projects focused on optimizing energy use in software systems."
+    items:
+      - name: "Zeus"
+        url: "https://ml.energy/zeus"
+        github_url: "https://github.com/ml-energy/zeus"
+        logo: "/open-source/logo/zeus.svg"
+        logo_scale: 70%
+        description: "Deep learning energy measurement and optimization"
+
+  - title: "Systems + AI"
+    description: "Combining AI with traditional systems research."
```

Comment:
This sounds more like AI for systems than systems for AI. I know we wanted to use Systems + AI to keep the door open, but I think we're mostly systems for AI.